motoko/hello-angular-motoko/canister_ids.json

```diff
@@ -0,0 +1,8 @@
+{
+  "motoko": {
+    "ic":"65yca-4yaaa-aaaak-qaqcq-cai"
+  },
+  "www": {
+    "ic": "zcp5t-tqaaa-aaaaj-aijea-cai"
+  }
+}
```

Comment:
This is your own `ic` deployment. I don't think you want those to be in the examples repo. You're welcome to link to them in the readme as a live example, however.

Reply:
Done.